tools/splash2xdmf.py

```diff
@@ -304,24 +305,24 @@ def create_xdmf_poly_attribute(dset, h5filename):
     poly.appendChild(attribute)
 
 
-def create_timestep_node(time):
+def create_interation_node(iteration):
     """
-    Return XDMF XML Time node 
+    Return XDMF XML iteration node 
     
     Parameters:
     ----------------
-    time: int
-          time step
+    iteration: int
+               iteration step
     Returns:
     ----------------
     return: Element
-            h5py XDMF Time node
+            h5py XDMF iteration node
     """
 
-    time_node = doc.createElement("Time")
-    time_node.setAttribute("TimeType", "Single")
-    time_node.setAttribute("Value", "{}".format(time))
-    return time_node
+    iteration_node = doc.createElement("Iteration")
+    iteration_node.setAttribute("IterationType", "Single")
```

Comment:
that won't work... both `Time` and `TimeType` are xdmf keywords...

(did you send in a blindly sed-replaced PR? ;) )